### PR TITLE
Use last version of kemal

### PR DESCRIPTION
**Pull request by waghanza**
Kemal has been release in `0.18.0`

http://kemalcr.com/blog/2017/02/11/kemal-0.18.0-released

### CODE REVIEW

**Comment by mperham** [comment]
Broken build

**Comment by rmosolgo** [comment]
> undefined constant MemoryIO (did you mean 'Memory')

Looks like a recent Crystal version [renamed it to `IO::Memory`](https://github.com/crystal-lang/crystal/pull/3563)

**Comment by neovintage** [comment]
@waghanza I managed to get sidekiq working based on your fixes for for baked_file_system. (Awesome, thank you) See my commits over here: https://github.com/neovintage/sidekiq.cr on master branch. 

Once you get past the `IO:Memory` stuff, a handful of issues popped up relative to language changes and kemal. `session` was removed from kemal and put in `kemal-session` shard. I used your code as a starting point and got all of the specs passing again. This is targeted at Crystal 0.21.0.

Let me know if you want any of the code.

**Comment by waghanza** [comment]
@neovintage in fact, I have commited on my branch and have a working CI 
https://travis-ci.org/waghanza/sidekiq.cr
but I do not understand why this `PR` is displayed as broken

File: README.md [hunk 1]
```diff
@@ -1,48 +0,0 @@
-# Sidekiq.cr
```
Comment (mperham) [inline README.md]: Why did you remove the readme?

Comment (waghanza) [inline README.md]: soory it was just a test (for travis-ci), I forget to remove my test

**Comment by mperham** [comment]
Thank you!
